### PR TITLE
Disable save button and change warning for too small vote_weight

### LINKED ISSUE #3058: [Meta] Prevent too small Vote weight and give better warnings
**Current behaviour:** 
If a participant is created with a too small vote_weight then the save-button is not disabled and if you try to save the participant-mask closes itself and an Error is thrown:
```
vote_weight must be bigger than or equal to 0.000001.
```
 If you try to create an account with too small vote_weight then the save button is disabled and the following warning is displayed:
If a user with vote_weight <= 0 is saved then the thrown error should be: 
```
Please select a vote weight bigger than zero.
```

**Reproduction:**
Participant: 
activate vote_weight in settings < participants
create a participant with a vote_weight < 0.000001

Account:
Put a vote_weight smaller than 0.000001 in the default_vote_weight box

Wanted behaviour: 
In the participant-create-mask: if the vote_weight is too small then the save button should be disabled

For the vote_weight/default_vote_weight input in the participant/account-mask:
If the input in the box is =< 0 then the following warning should be shown:
 ```
vote_weight must be greater than 0.
```

If the input in the box is  >0 but <0.000001 then the following warning should be thrown:
```
vote_weight must be greater than or equal to 0.000001.
```

### PR DESCRIPTION
**Pull request by Elblinator**
closes #3058

### CODE REVIEW

File: client/src/app/site/pages/organization/pages/accounts/pages/account-detail/components/account-detail/account-detail.component.ts
Comment (bastianjoel) [inline client/src/app/site/pages/organization/pages/accounts/pages/account-detail/components/account-detail/account-detail.component.ts:1]: I don't think the changes in this file are needed.

File: client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/pages/participant-detail-manage/components/participant-create-wizard/participant-create-wizard.component.ts [hunk 1]
```diff
@@ -92,6 +100,7 @@ export class ParticipantCreateWizardComponent extends BaseMeetingComponent imple
     public personalInfoFormValue: any = {};
     public formErrors: { [name: string]: boolean } | null = null;
     public groupsObservable: Observable<ViewGroup[]> | null = null;
+    public form: UntypedFormGroup;
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/pages/participant-detail-manage/components/participant-create-wizard/participant-create-wizard.component.ts:103]: `form` seems to be defined by the `ng-template`. So no need to add this here.

**Review by bastianjoel — CHANGES_REQUESTED** [review]

**Review by bastianjoel — APPROVED** [review]

File: client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.html [hunk 2]
```diff
@@ -135,6 +135,12 @@ <h2>{{ 'Meeting specific information' | translate }}</h2>
                         placeholder="{{ 'Vote weight' | translate }}"
                         formControlName="vote_weight"
                     />
+                    <mat-error *ngIf="isVoteWeightError && form.controls['vote_weight'].value > 0" translate>
+                        Please select a vote weight greater than or equal 0.000001
```
Comment (emanuelschuetze) [inline client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.html:139]: "... or equal _to_ ..." -> missing word

**Comment by emanuelschuetze** [comment]
New translations of upstream/main contains also the new strings of this PR. But please fix the remark (missing "to") which I've fixed locally for my translation work.

**Review by MSoeb — APPROVED** [review]

**Pull request merged**